path_planning_rate : 1
fusion_map_queue_size : 10
dest_queue_size : 10
traffic_light_queue_size : 10
localize_queue_size : 10
t_sample : 0.02
t_horizon : 6
dest_x : 61.99
dest_y : 10.63
debug_flag : 0

ego_length : 4.4
ego_width : 1.8
ego_a : 4.9
ego_max_v : 30

traffic_propogation_resolution: 0.2

static_obstacle_map_center_x: 499
static_obstacle_map_center_y: 499

prepare/threshold_lane_confidence : -0.1
prepare/threshold_line_combination : 0.1
prepare/interp_delta_length : 1
prepare/interp_delta_angle : 0.1
prepare/localize_timedelta : 100
prepare/traffic_light_timedelta : 500

feature_extraction/min_dist : 10
feature_extraction/min_width : 0.3

maneuver/lane_state_num : 5
maneuver/intersection_state_num : 4
maneuver/lane_transition_matrix : [0, 1, 2, 3, 5, 4, 0, 0, 0, 0, 0, 1, 0, 3, 5, 0, 1, 2, 0, 0, 0, 0, 0, 0, 0]
maneuver/intersection_transition_matrix : [0, 7, 6, 8, 0, 0, 0, 8, 4, 0, 6, 0, 9, 7, 6, 8]
maneuver/lane_state_order : [0, 2, 1, 3, 4]
maneuver/intersection_state_order : [0, 1, 2, 3]
maneuver/feature_order_R : [0, 0, 1, 2, 3, 17, 18, 19]
maneuver/feature_order_E : [0, 7, 4, 5, 6, 8, 10, 9, 11, 12, 13, 16, 21, 22, 23, 20, 24, 25, 14, 15]

candidate_trajectory/use_gpu: false 
candidate_trajectory/thread_num : 10
candidate_trajectory/record_trajectory : false
candidate_trajectory/num_d : 5
candidate_trajectory/num_t : 5
candidate_trajectory/num_s : 5 
candidate_trajectory/min_t : 1
candidate_trajectory/max_t : 5
candidate_trajectory/min_s : 1
candidate_trajectory/max_s : 20
candidate_trajectory/v_ref_min_ratio : 0.5
candidate_trajectory/v_ref_max_ratio : 1
candidate_trajectory/acceleration_nomalization : 20
candidate_trajectory/jerk_nomalization : 20
candidate_trajectory/curvature_nomalization : 0.1
candidate_trajectory/coefficient_s_val : 0.1
candidate_trajectory/coefficient_s_deriv1 : 1
candidate_trajectory/coefficient_s_deriv2 : 1
candidate_trajectory/coefficient_d_val : 10
candidate_trajectory/coefficient_d_deriv1 : 1
candidate_trajectory/coefficient_d_deriv2 : 1
trajectory_selection/velocity_ratio : 1
trajectory_selection/acceleration_ratio : 1
trajectory_selection/jerk_ratio : 1
trajectory_selection/position_ratio : 10
trajectory_selection/curvature_ratio : 5
trajectory_selection/similarity_ratio : 1
trajectory_selection/risk_barrier : 0.9

risk_helper/ori_x : 50
risk_helper/ori_y : 50
risk_helper/x_lhs_bound : 0
risk_helper/x_rhs_bound : 499
risk_helper/y_lhs_bound : 0
risk_helper/y_rhs_bound : 749

traffic_propagation/certain_trajectory_start_time : 0
traffic_propagation/certain_trajectory_end_time : 2000
traffic_propagation/certain_trajectory_step : 100
traffic_propagation/resolution : 0.2

global_planning/map_file : map_data/map.xml
global_planning/default_speedlimit : 20
global_planning/default_cross_time : 30 
global_planning/manual: 1
global_planning/section_ids: [16, 4, 5, 7, 17]

